packages/ui/src/lib/table/Table.svelte

```diff
@@ -39,7 +39,7 @@ $: selectedItemsNumber = row.info.selectable
   : 0;
 
 // do we need to unselect all checkboxes if we don't have all items being selected ?
-$: selectedAllCheckboxes = row.info.selectable
+let selectedAllCheckboxes = row.info.selectable
```

Comment:
This breaks the behaviour described in the comment above: if you select all the checkboxes individually, the top 'all' checkbox should be automatically checked. With this change, this does not happen anymore